src/Text/Pandoc/Readers/HTML.hs

```diff
@@ -623,13 +623,49 @@ pPlain = do
      then return mempty
      else return $ B.plain contents
 
-pPara :: PandocMonad m => TagParser m Blocks
-pPara = do
+-- Helper function for pPara when significant attributes are present
+pParaWithWrapper :: PandocMonad m => Attr -> TagParser m Blocks
+pParaWithWrapper (ident, classes, kvs) = do
+  guardEnabled Ext_native_divs -- Ensure native_divs is enabled for this behavior
+  pInhalt <- trimInlines <$> pInTags "p" inline
+  (do guardDisabled Ext_empty_paragraphs
+      guard (null pInhalt)
+      return mempty) <|> do
+    let wrapperAttr = ("wrapper", "1")
+    let alignValue = lookup "align" kvs
+    let otherKVs = filter (\(k,_) -> k /= "align") kvs
+    let validAlignKV = case alignValue of
+                         Just algn | algn `elem` ["left","right","center","justify"] -> [("align", algn)]
+                         _ -> []
+    let finalKVs = wrapperAttr : (validAlignKV ++ otherKVs)
```

Comment:
What is the motivation for treating the "align" attribute specially in this way?